tests/test_pasqal.py

```diff
@@ -208,16 +208,6 @@ def test_submit(fixt, parametrized, emulator, seq, mock_job):
         )
     )
 
-    job_params[0]["runs"] = 1
-    with pytest.raises(RuntimeError, match="Failed to find job ID"):
-        # Job runs don't match MockJob
-        fixt.pasqal_cloud.submit(
-            seq,
-            job_params=job_params,
-            emulator=emulator,
-            config=config,
-        )
-
```

Comment:
Why is this test no longer relevant ?

Reply:
Because we no longer try to determine the jobs order (the code where this was being raised was deleted)

Reply:
Okay, that was my guess... So it's no longer an issue that the number of runs in job_params is different than number of runs in "mock_job" ? The two issues seemed independent to me, but I am not I understood the test in the first place...

Reply:
That manual change was just a way for the job params to not match anymore and thus trigger the fail to find the job ID.